app/src/main/res/layout/feed_item_list_fragment.xml

```diff
@@ -48,12 +48,19 @@
         </com.google.android.material.appbar.CollapsingToolbarLayout>
     </com.google.android.material.appbar.AppBarLayout>
 
-    <de.danoeh.antennapod.view.EpisodeItemListRecyclerView
-            android:id="@+id/recyclerView"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:paddingHorizontal="@dimen/additional_horizontal_spacing"
-            app:layout_behavior="@string/appbar_scrolling_view_behavior" />
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+        android:id="@+id/swipeRefresh"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingHorizontal="@dimen/additional_horizontal_spacing"
```

Comment:
I have not tried it yet but I think this padding should be applied to the recyclerView, not the swipe layout. Otherwise, the scroll bar on big tablets will not be on the edge of the screen.

Reply:
I am not sure but I think this does not make any difference as the recyclerview matches the parent or is the padding attribute handled differently in a recyclerview than it is in a listview?

Reply:
It does make a difference :) The recyclerView's scroll bar is then influenced by the padding. See screenshot below. Before, the scroll bar was on the right side of the screen.

![image](https://user-images.githubusercontent.com/5811634/94945823-8cb5b680-04db-11eb-90d2-783fd16f7e7b.png)

Reply:
Well, I always learn something new. :wink: